src/fluree/db/reasoner/owl_datalog.cljc

```diff
@@ -607,11 +1051,14 @@
 (defmethod to-datalog ::cax-eqc
   [_ inserts owl-statement all-rules]
   (let [c1 (util/get-id owl-statement) ;; the class which is the subject
+        equiv-class-val (get owl-statement const/iri-owl:equivalentClass)
+        unwrapped (util/unwrap-list equiv-class-val)
+        ;; Ensure unwrapped is always a sequence for group-by
+        unwrapped-seq (if (sequential? unwrapped) unwrapped [unwrapped])
```

Comment:
We have `util/sequential` for this, though I think it would be better for `unwrap-list` can be refactored to always return a sequence, since other callsites assume that it does.